src/partest/scala/tools/partest/IcodeComparison.scala

```diff
@@ -49,7 +49,7 @@ abstract class IcodeComparison extends DirectTest {
   /** Compile the test code and return the contents of all
    *  (sorted) .icode files, which are immediately deleted.
    *  @param arg0 at least one arg is required
-   *  @param args must include -Xprint-icode:phase
+   *  @param args must include -Vprint-icode:phase
```

Comment:
Should I feel bad that I can't locate this option anywhere? I have a vague memory about a mechanism for testing, but I don't have time to track it down.